beta/src/content/learn/start-a-new-react-project.md

```diff
@@ -27,13 +27,13 @@ React is a library that lets you organize UI code by breaking it apart into piec
 
 ## Getting started with a minimal toolchain {/*getting-started-with-a-minimal-toolchain*/}
 
-If you're **learning React,** we recommend [Create React App.](https://create-react-app.dev/) It is the most popular way to try out React and build a new single-page, client-side application. It's made for React but isn't opinionated about routing or data fetching.
+If you're **learning React,** we recommend the [Vite React template.](https://vitejs.dev/guide/) It's the simplest way to try out React and build a new single-page, client-side application. Vite works with other frameworks and isn't opinionated about routing or data fetching.
```

Comment:
Why not just adding a couple of solutions to pick from 😄

Reply:
well, due to this section being related to learning react as first timer – we're talking about people with possibly 0 experience and we cannot expect them to dig into it nor understand differences

Reply:
especially when CRA itself is not _that_ bad - it's just unmaintained and can be painful couple days later due to lack of configuration (hello @tailwindcss) and at that point in time person can just say "f it - i'm outta here" and never come back

Reply:
> especially when CRA itself is not _that_ bad - it's just unmaintained

I would argue that alone makes it bad. 1.5k open issues, no merges in months, it's a dead project

It's also old tech. We don't do things that way anymore why point new dev to using just for the sake of "it was always this way"?

Reply:
> That said, I agree with @devongovett that the documentation should be more nuanced here, because it does not only serve beginners. There should be a default option for beginners (and, as I did for my book, I am in favor for Vite here, because it does not come with much overhead (e.g. Next.js with SSR, NX/Turborepo for Monorepos) and, in my subjective opinion, it was one of the causes why people migrated heavily away from CRA (which didn't happen at this extreme rate when Parcel was released, because, I guess, it was just ahead of its time). Anyway, as said, the documentation could offer a default for beginners, followed by a list of alternatives (e.g. Parcel, Next, Remix, Astro, Turborepo) in their respective fields of expertise (e.g. SSR, Islands, Monorepos) for more intermediate developers.

Have you taken a look at nx standalone-react which isn't a monorepo and a strip back version just like CRA but with NX caching, and migrations

Reply:
I'd just add Vite as the main one and list a couple of alternatives to dig deeper for the interested ones 😉

Reply:
Why not show more options instead of just one?

Reply:
I've had so many issues with Vite as a project. Personally I'm surprised to the community so eager to recommend it.

Reply:
@nstanard That's because as imperfect as Vite is, it's still actively developed.

The mountains of issues continuing to pile up (many with PR fixes that have been ignored for a very long time) can never be resolved at this point without a full rewrite.

Reply:
This PR updates a section of the docs that only a beginner would need. CRA is still a perfectly acceptable tool for a beginner to start learning. I have a hard time seeing this change as necessary. It comes off as a publicity tactic by people in the Vite community.

Reply:
80% of the concern would be alleviated if they simply added a note that explicitly stated CRA is a learning tool that shouldn't be used for production.

Reply:
> It comes off as a publicity tactic by people in the Vite community.

What you call a publicity tactic, I call sharing with my brothers in code the merciful word of the one true boilerplate tool to rule them all: Vite 🫡

Reply:
@wesleycoder I fully agree 🙂 That's why there would be one thing recommended. And for the rest there could a link to another page that explores other options ;)

Second thing I've just realized. I remember when I was starting with React and I was using CRA.
The problem was that I was using an old MacBook Pro from 2011 with 4GB of RAM.
Setting up a CRA project and running it with another app (for example simple backend) in the background was painful and slow. The reloading of the project sometimes was taking ages. And it was just a couple of years ago.

Then I tried Vite. On that very same laptop. Woah it was a breeze ❤️ Project setup took couple of seconds (compared to sometimes 10minutes for CRA, ofc on the device I mentioned) and you could just feel how lightweight of a solution that is if you just want to build a React application or just play with some new stuff.

So that could be an argument for Vite. Not everybody has the latest spec for their devices 😄